CS410_Assignment2_CodeReview/src/sensormain/SensorApplication.java

```diff
@@ -0,0 +1,205 @@
+/*
+ * Lauren Kearney
+ * CS 410
+ * 
+ * three panel GUI for Sensor application
+ * 
+ * */
+
+package sensormain;
+
+import java.awt.Color;
+import java.awt.Dimension;
+import java.awt.GridLayout;
+import java.util.ArrayList;
+import java.util.List;
+
+import java.awt.*;
+import javax.swing.*;
+
+import javax.swing.JFrame;
+import javax.swing.JLabel;
+import javax.swing.JPanel;
+import javax.swing.JProgressBar;
+import javax.swing.border.TitledBorder;
+
+import sensor.PressureSensor;
+import sensor.RadiationSensor;
+import sensor.TemperatureSensor;
+
+public class SensorApplication extends JFrame {
+	
+	public PressureSensorAdapter pressure = new PressureSensorAdapter(new PressureSensor());
+	public RadiationSensorAdapter radiation = new RadiationSensorAdapter(new RadiationSensor());
+	public TemperatureSensorAdapter temperature = new TemperatureSensorAdapter(new TemperatureSensor());
+	
+	public SensorApplication() {
+		

```

Comment:
Recommend calling values and saving them to a variable to be used later. It returns a random value each time is called, and is used to update the status as well. 
`double pressureValue = pressure.value()`
`double radiationValue = radiation.value()`
`double temperatureValue = temperature.value()`